index.md

```diff
@@ -9,6 +9,189 @@ notoc: true
 Docker packages your app with its dependencies, freeing you from worrying about
 your system configuration, and making your app more portable.
 
+{% if site.edge == true %}
+## Docker CE Edge
+
+The Docker CE Edge channel provides monthly releases which allow you to try
+new features of Docker and verify bug fixes quickly. Edge releases are only
+supported for one month, including security updates and high priority bug fixes
+only. When a new release becomes available in the Docker CE Edge channel,
+previous releases stop receiving updates. Commercial support is not available
+for Docker CE.
+
+For information about all Docker release channels and expectations about
+support, see [Docker channels](/engine/installation/#docker-channels)
+
+This page lists features that are only available in Docker CE Edge releases.
+Where applicable, the API and CLI reference documentation has been updated to
+reflect these features, but **full documentation for a given feature may not be
+available until a Docker CE Stable release incorporates the feature**.
+
+### Docker CE Edge new features
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#1704">17.04</a></li>
+  <!--<li><a data-toggle="tab" href="#1705">17.05</a></li>-->
+</ul>
+<div class="tab-content">
+<div id="1704" class="tab-pane fade in active">
```

Comment:
Would be cool if we could template this, and use include files (`tabs/edge-17.04.md`, and it automatically generate the boiler plating for the tabs) 🤔